src/Components/Categories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// eslint-disable-next-line
+import { checkStatus } from "../redux/categories/categories";
+
+export default function Categories() {
+  const status = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+  // console.log(status);
```

Comment:
- [ ] Kindly remove this `console.log` statement from this file. Using `console.log` is good for debugging but it should be removed after use. This can prevent the leakage of sensitive information. 👍 😎